content/blog/announcing-osm-v1.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Announcing OSM v1.0.0"
+slug: "announcing-osm-v1"
+authorname: "@phillipgibson & @nojnhuh"
```

Comment:
This is displayed literally, so you probably want this to be listed as "Phillip Gibson & Jon Huhn" instead of handles that won't be linked to.

Reply:
Updated.